### ISSUE TITLE
Globbing: Ignore VCS directories by default

### ISSUE THREAD

**Issue by underyx** [coala/coala#3656]
We can probably reuse the heuristics of [`ag`](https://github.com/ggreer/the_silver_searcher) for this.

**Comment by underyx** [comment]
Like in #3655:

>I can do this on Friday, if someone else can do it earlier, go ahead and take it.

**Comment by Makman2** [comment]
what about putting `.git/*` and others into our default coafile?

**Comment by underyx** [comment]
But then wouldn't the ignore disappear once someone sets their own `ignore` directories?

**Comment by Makman2** [comment]
yeah we have currently this problem. Already discussed inside https://github.com/coala/coala/issues/3131 how to eventually tackle this^^

**Comment by jayvdb** [comment]
Re `ag`, see https://github.com/coala/coala/issues/3131#issuecomment-382418432
